breakwater/src/statistics.rs

```diff
@@ -91,22 +79,21 @@ pub struct Statistics {
 }
 
 impl StatisticsInformationEvent {
-    fn save_to_file(&self, file_name: &str) -> Result<(), Error> {
+    fn save_to_file(&self, file_name: &str) -> eyre::Result<()> {
         // TODO Check if we can use tokio's File here. This needs some integration with serde_json though
         // This operation is also called very infrequently
-        let file = File::create(file_name).context(CreateStatisticsSaveFileSnafu {
-            save_file: file_name.to_string(),
-        })?;
-        serde_json::to_writer(file, &self).context(SerializeStatisticsSnafu)?;
+        let file = File::create(file_name).context(format!(
+            "failed to create statistics save file at {file_name}"
+        ))?;
+        serde_json::to_writer(file, &self).context("failed to write statistics to file")?;
```

Comment:
optional: It would be great if you could mention the file name here as well, so that e.g. permission problems can be sorted out more easily